src/main/java/net/javadiscord/javabot/data/h2db/commands/ExportSchemaSubcommand.java

```diff
@@ -1,48 +1,61 @@
 package net.javadiscord.javabot.data.h2db.commands;
 
+import com.dynxsty.dih4jda.interactions.commands.SlashCommand;
+import net.dv8tion.jda.api.Permission;
 import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
 import net.dv8tion.jda.api.interactions.commands.OptionMapping;
-import net.dv8tion.jda.api.requests.restaction.interactions.ReplyCallbackAction;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
 import net.javadiscord.javabot.Bot;
-import net.javadiscord.javabot.command.interfaces.SlashCommand;
+import net.javadiscord.javabot.util.ExceptionLogger;
 
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.sql.SQLException;
 
 /**
- * This subcommand exports the database schema to a file, and uploads that file
+ * <h3>This class represents the /db-admin export-schema command.</h3>
+ * This Subcommand exports the database schema to a file, and uploads that file
  * to the channel in which the command was received.
  */
-public class ExportSchemaSubcommand implements SlashCommand {
+public class ExportSchemaSubcommand extends SlashCommand.Subcommand {
 	private static final Path SCHEMA_FILE = Path.of("___schema.sql");
```

Comment:
I don't think that's really necessary. I mean, yeah, sure, but the current solution always worked since the command was first ever introduced by Andrew, so I think this can stay.

Reply:
This was just a suggestion, we don't have to do that.